### PR TITLE
Cloud sync UI Changes

### PR DESCRIPTION
**Pull request by livcheerful**
- Add the Cloud Card to projects list if you're not signed in
- Add custom icon / sign in text to the header

based on Design office hour changes

looks like this!

image

### CODE REVIEW

File: theme/common.less [hunk 1]
```diff
@@ -1938,6 +1938,21 @@ div.simframe.ui.embed {
...
+div.signin-button {
+    display: flex;
+    align-items: center;
+
+    .text {
+        margin-right: .5rem;
+    }
+}
+
+@media only screen and (max-width: @largestTabletScreen) {
```
Comment (livcheerful) [inline theme/common.less:1950]: This is to keep the icon floating right. The default rules are lost bc I added an extra div around the whole sign in button 

image
 vs 
image

Comment (livcheerful) [reply theme/common.less:1950]: (in not desktop view)

File: theme/themes/pxt/views/card.overrides [hunk 2]
```diff
@@ -152,6 +152,17 @@ a.ui.card:focus,
...
     background: fade(@primaryColor, 90%) !important;
 }
+
+.ui.card.link.cloudprojectscard {
+    background: @cloudProjectCardBackground!important;
+}
+
+@media only screen and (min-height: tallEditorBreakpoint), (min-width: @computerBreakpoint) {
+    .ui.card.link.cloudprojectscard .icon {
+        margin-left: -1rem
```
Comment (livcheerful) [inline theme/themes/pxt/views/card.overrides:162]: for some reason the new cloud svg floats right and looks silly when large. This bumps it to the left until it hits the smaller icon size, where it looks properly centered

without the extra rule:
image

Comment (livcheerful) [reply theme/themes/pxt/views/card.overrides:162]: with:
image

File: webapp/src/projects.tsx [hunk 3]
```diff
@@ -747,6 +748,13 @@ export class ProjectsCarousel extends data.Component<ProjectsCarouselProps, Proj
                         <span className="header">{lf("New Project")}</span>
                     </div>
                 </div> : undefined}
+                {showCloudProjectsCard ? <div role="button" className="ui card link buttoncard cloudprojectscard" title={lf("Sign in to see cloud projects")}
```
Comment (eanders-MS) [inline webapp/src/projects.tsx:751]: super minor suggestion: using `{showCloudProjectCard && ...}` would make this statement slightly more compact.

File: theme/themes/pxt/views/card.overrides (hunk 2 above)
Comment (eanders-MS) [reply theme/themes/pxt/views/card.overrides:162]: How does the card look on mobile? Wondering if "Cloud Projects" wraps, or doesn't but should...

File: theme/themes/pxt/globals/site.variables [hunk 4]
```diff
@@ -176,6 +176,7 @@
 @sidebarTutorialColor: @lightGrey;
 
 @homeScreenBackground: #ecf0f1;
+@cloudProjectCardBackground: #58AFBA;
```
Comment (eanders-MS) [inline theme/themes/pxt/globals/site.variables:179]: This css variable name is very specific. Might be better to make it categorical, like `@simulatorBackground` (or whatever makes sense).

**Review by eanders-MS — COMMENTED** [review]
Looks good! Left a few minor comments/questions.

File: theme/themes/pxt/views/card.overrides (hunk 2 above)
Comment (livcheerful) [reply theme/themes/pxt/views/card.overrides:162]: it doesn't, I'll add it

image

File: theme/themes/pxt/globals/site.variables (hunk 4 above)
Comment (livcheerful) [reply theme/themes/pxt/globals/site.variables:179]: hmmm yeah I was thinking about that, but I was hesitant to give it a grand name when the color is just used once for now.

**Review by eanders-MS — APPROVED** [review]
LGTM!

**Pull request merged**